common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -9,8 +9,12 @@ fun debugLog(message: String) {
     }
 }
 
-fun log(message: String) {
-    Log.w("[Purchases] - INFO", message)
+fun infoLog(message: String) {
+    Log.i("[Purchases] - INFO", message)
+}
+
+fun warnLog(message: String) {
+    Log.w("[Purchases] - WARN", message)
 }
 
 fun errorLog(message: String) {
```

Comment:
@aboedo Noticed in iOS logger this would be on by default. Before removing the `Config.debugLogsEnabled`, is `debugLog` the only function this flag would be checked?

Reply:
I think always showing error logs is a good idea, and it's consistent with how most loggers consider the hierarchy between log levels